tools/fpgabist/bist/bist_afu.c

```diff
@@ -256,18 +256,17 @@ int main(int argc, char *argv[])
         /* Perform BIST Check */
 	printf("Running BIST Test\n");
         uint64_t data = 0;
-        uint64_t fpga_res = 0;
         double count = 0;
         unsigned int bist_mask = ENABLE_DDRA_BIST;
         unsigned int ddra_bist_result = 0;
 
 	res = fpgaWriteMMIO32(accelerator_handle, 0, DDR_BIST_CTRL_ADDR, bist_mask);
 
-        fpga_res = fpgaReadMMIO64(accelerator_handle, 0, DDR_BIST_CTRL_ADDR,&data);
+        fpgaReadMMIO64(accelerator_handle, 0, DDR_BIST_CTRL_ADDR,&data);
 	while((CHECK_BIT(data,27) != 0x08000000)){
-	  printf("Enable Test: reading result #%f: %04x\n", count, data);
+	  printf("Enable Test: reading result #%f: %04x\n", count, (unsigned int)data);
```

Comment:
you should probably use `%lx` in printf formatting rather than using a cast here (in general avoid unnecessary casting and don't use casting to silence warnings except in rare cases).